src/components/input-time-picker/input-time-picker.tsx

```diff
@@ -502,18 +502,17 @@ export class InputTimePicker
 
   private async loadDateTimeLocaleData(): Promise<void> {
     const { effectiveLocale } = this;
+    let dayjsLocale = effectiveLocale.toLowerCase();
```

Comment:
could `effectiveLocale` be null? If so should there be a check?

Reply:
It should always default to `en` if this is sent in as a null value, but couldn't hurt to do a null check here.

Reply:
Under normal circumstances, `effectiveLocale` will never be null, so need for the null check.